src/main/java/net/dv8tion/jda/internal/handle/VoiceStateUpdateHandler.java

```diff
@@ -183,5 +188,10 @@ else if (channel == null)
                                 member, oldChannel));
             }
         }
+        if (voiceInterceptor != null)
```

Comment:
This implicitly means "If this is for the selfUser _and_ we have a voiceInterceptor.
Instead, we should be explicit in checking `selfUser && voiceInterceptor != null`.

As such this retrieval should not be selfUser guarded: https://github.com/DV8FromTheWorld/JDA/pull/860/files#diff-daf701d8ea1b39bcdb9e6e159a9c79fbR103